bolt-ons/gps-bolt-on/GPS_2_CAN.ino

```diff
@@ -1,49 +1,87 @@
 #include <SoftwareSerial.h>
 #include <TinyGPS++.h>
+#include <SPI.h>
+#include "mcp_can.h"
```

Comment:
Hi, please use CAN1, which uses the internal CAN controller of the ESP32-S3.  Here you're using the MCP2515 (CAN2). 

You can find the example of CAN1 in the ping pong demo. Since you're using only CAN1, you can remove any reference to MCP2515 and CAN2.